audio/abcMIDI/Portfile

```diff
@@ -21,17 +18,23 @@ long_description    ABC is a text-based music notation language. Abc2midi is a p
 homepage            https://ifdo.ca/~seymour/runabc/top.html
 master_sites        https://ifdo.ca/~seymour/runabc/
 
-worksrcdir          abcmidi
+checksums           rmd160  ba8b24b4c4ac80a0c8eae38b90e7c1e689f58fa4 \
+                    sha256  497a5a6ef27fa46d69d69b82a2769f37507acd52b92857b0f0b628dcfc8d9b14 \
+                    size    603543
 
 use_zip             yes
 
 post-extract {
-    file attributes ${worksrcpath}/configure -permissions a+x
+    system "chmod a+x ${worksrcpath}/*"
```

Comment:
Is it really necessary to give all files in the directory execute permissions ? What files other than configure need it ?
We generally prefer the file attributes method instead of using a system call.

Reply:
I fixed it, if I don't confuse anything in Tcl, now only directories are given execution permissions.